Robust.Shared/GameObjects/EntityEventBus.Directed.cs

```diff
@@ -12,6 +12,8 @@ public interface IDirectedEventBus
         void RaiseLocalEvent<TEvent>(EntityUid uid, TEvent args, bool broadcast = true)
             where TEvent:EntityEventArgs;
 
+        public void RaiseLocalEvent(EntityUid uid, EntityEventArgs args, bool broadcast = true);
```

Comment:
```suggestion
        void RaiseLocalEvent(EntityUid uid, EntityEventArgs args, bool broadcast = true);
```

Unnecessary access modifiers in interfaces annoy me.